SQL/SQL.sublime-syntax

```diff
@@ -96,8 +96,14 @@ contexts:
         1: keyword.operator.logical.sql
         2: keyword.operator.logical.sql
         3: constant.language.sql
-    - match: (?i:\b(and|or|like|having|exists|between|in)\b)
+    - match: (?i:\b(and|or|having|exists|between|in)\b)
       scope: keyword.operator.logical.sql
+    - match: (?i:\blike\b)
+      scope: keyword.operator.logical.sql
+      push: like_string
+    - match: (?i:\bescape\b)
+      scope: keyword.operator.sql
```

Comment:
Word like operators may wanted to be highlighted differently from symbol like operators. 

I have Breackets Color Scheme in mind for instance, which might want to highlight ESCAPE like the other keywords (in blue).

![grafik](https://user-images.githubusercontent.com/16542113/132252006-630d5c38-90de-492f-99cd-de0bc6cedb29.png)

A third-level scope may help. Not sure which one as I haven't seen such a use case yet.

Reply:
I've made it a `keyword.operator.word` for now in case it helps, but happy to revise.

Reply:
Let's go with it for now.